cassovary-core/src/main/scala/com/twitter/cassovary/algorithms/centrality/Centrality.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2014 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.twitter.cassovary.algorithms.centrality
+
+trait Centrality {
+
+  def run: Array[Double]
```

Comment:
And maybe apply as the method name? That would be consistent with other algorithms we have (ex. PageRank)